### PR TITLE
Feat/firefox

### PR DESCRIPTION
**Pull request by hz002**

### CODE REVIEW

**Comment by carterlasalle** [comment]
#528 Is the related issue

File: src/constant/index.ts [hunk 1]
```diff
@@ -464,7 +466,7 @@ export const INTERNAL_REQUEST_SESSION = {
 
 export const INITIAL_OPENAPI_URL = 'https://api.rabby.io';
 
-export const INITIAL_TESTNET_OPENAPI_URL = 'https://api.testnet.rabby.io';
+export const INITIAL_TESTNET_OPENAPI_URL = 'https://alpha.rabby.io';
```
Comment (vvvvvv1vvvvvv) [inline src/constant/index.ts:469]:
```suggestion
export const INITIAL_TESTNET_OPENAPI_URL = 'https://api.testnet.rabby.io';
```

**Comment by Eleleth77** [comment]
does this mean it is compatible on FF? Not seeing it in the addon store, nor the main rabby website 🤔
Thank you all so much, FF community really appreciates it 🫡

**Comment by Polve** [comment]
I can't find it on the extensions list of firefox too: is there another way to install it?

**Comment by carterlasalle** [comment]
Thanks to everyone who upvoted! It's been a grind getting this done, but because of you, we have it now :)
🥇

**Comment by elboletaire** [comment]
Now we need it to be properly detected, couldn't find a single provider allowing me to use rabby.

Appkit for example has support for rabby, but right now redirects to the chromestore without properly detecting rabby on the browser.

imatge

**Comment by vvvvvv1vvvvvv** [comment]
We already upload a fix version to Firefox 5 days ago, but Firefox review is slow, after they approve the version, the issue will be fixed

**Comment by elboletaire** [comment]
> We already upload a fix version to Firefox 5 days ago, but Firefox review is slow, after they approve the version, the issue will be fixed

Actually, it was accepted the very same day [according to the Mozilla Add-ons page](https://addons.mozilla.org/ca/firefox/addon/rabby-wallet/), but I guess my browser hadn’t updated it yet. I just tried it out, and it works perfectly now!

I’m so happy to be able to use Rabby on Firefox now 🥳!

**Comment by zotabee** [comment]
Looks like the firefox addon is gone from Mozilla store. Also no sign of it on rabby.io. 🤔

**Comment by eboye** [comment]
No Firefox, no Linux ... well, I don't know how I'm supposed to use it? :D

**Comment by Polve** [comment]
WTH happened?
I was just wondering why I had no more updates since long time?

**Pull request merged**